From be579a50b815fb03ad312eed662a28850ee4c5b7 Mon Sep 17 00:00:00 2001
From: Chris Short <chris@chrisshort.net>
Date: Mon, 18 Jan 2016 09:11:04 -0500
Subject: [PATCH 16/28] BUG/MINOR: examples: Fixing haproxy.spec to remove
 references to .cfg files

Building RPMs from the provided haproxy.spec fails due to references to
config files that do not exist.

I should point out, this patch will inevitably create an empty /etc/haproxy
dir and I'm not sure that's desirable/intended.
(cherry picked from commit bb107185d1985eb7a45ecb2fe0119b8e8f9b3233)
---
 examples/haproxy.spec | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/examples/haproxy.spec b/examples/haproxy.spec
index 224597d..f09a529 100644
--- a/examples/haproxy.spec
+++ b/examples/haproxy.spec
@@ -44,7 +44,6 @@ risking the system's stability.
 %{__install} -d %{buildroot}%{_mandir}/man1/
 
 %{__install} -s %{name} %{buildroot}%{_sbindir}/
-%{__install} -c -m 644 examples/%{name}.cfg %{buildroot}%{_sysconfdir}/%{name}/
 %{__install} -c -m 755 examples/%{name}.init %{buildroot}%{_sysconfdir}/rc.d/init.d/%{name}
 %{__install} -c -m 755 doc/%{name}.1 %{buildroot}%{_mandir}/man1/
  
@@ -67,12 +66,11 @@ fi
 
 %files
 %defattr(-,root,root)
-%doc CHANGELOG README examples/*.cfg doc/architecture.txt doc/configuration.txt doc/intro.txt doc/management.txt doc/proxy-protocol.txt
+%doc CHANGELOG README doc/architecture.txt doc/configuration.txt doc/intro.txt doc/management.txt doc/proxy-protocol.txt
 %doc %{_mandir}/man1/%{name}.1*
 
 %attr(0755,root,root) %{_sbindir}/%{name}
 %dir %{_sysconfdir}/%{name}
-%attr(0644,root,root) %config(noreplace) %{_sysconfdir}/%{name}/%{name}.cfg
 %attr(0755,root,root) %config %{_sysconfdir}/rc.d/init.d/%{name}
 
 %changelog
-- 
2.4.10

